mattermost/DEBIAN/postrm

```diff
@@ -1,7 +1,19 @@
 #!/usr/bin/env bash
 
+set -e
+
 case "$1" in
     remove)
-        systemctl daemon-reload
-        ;;
+
+    [ -z "$MATTERMOST_USER" ] && MATTERMOST_USER="mattermost"
```

Comment:
same as I mentioned above

Reply:
We are trying to perform the following:
If MATTERMOST_USER is null (not set) we set to to `mattermost`
Same for MATTERMOST_GROUP

Reply:
same as above

Reply:
Tested it, seems to work as expected.

What do we gain with the proposal?
Wouldn't you agree that it is a more readable form? :
[ -z "$MATTERMOST_USER" ] && MATTERMOST_USER="mattermost"
vs
MATTERMOST_USER=${MATTERMOST_USER:-mattermost}

The latter requires some bash insights.

Would vote in favor of readability against saving a couple of chars.

Let me know what do you think